relay/server.go

```diff
@@ -159,6 +159,7 @@ func NewServer(
 		authenticator:    authenticator,
 		replayGuardian:   replayGuardian,
 		metrics:          relayMetrics,
+		chainReader:      chainReader,
```

Comment:
This might be a bug lol

Reply:
...?? how/where does relay use chainReader. and how was it working without this?